.jenkins/Jenkinsfile

```diff
@@ -155,7 +155,7 @@ def win2016LinuxElfBuild(String version, String compiler, String build_type) {
                 dir('build') {
```

Comment:
If you add this here, it should unblock CI for this build:

powershell 'Copy-Item -Recurse C:\\openenclave\\prereqs\\nuget ${env:WORKSPACE}\\prereqs'